examples/state_versions/main.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package main
+
+import (
+	"context"
+	"crypto/md5"
+	"fmt"
+	"log"
+	"os"
+
+	tfe "github.com/hashicorp/go-tfe"
+)
+
+func main() {
+	ctx := context.Background()
+	client, err := tfe.NewClient(&tfe.Config{
+		RetryServerErrors: true,
+	})
```

Comment:
Nice example 👍 -- If you add the example to `example_test.go` it will also show the example in GoDocs, which I think is preferrable to looking at the Github example.